README.md

```diff
@@ -242,6 +242,63 @@ Autoscaling rules are applied automatically during the next deploy, though may b
 ```shell
 dokku scheduler-kubernetes:autoscale-rule-apply $APP PROC_TYPE
 ```
+### Persistent Volume Claims (pvc) 
+
+Pods use pvcs as volumes.  For volumes that support multiple access modes, the user specifies which mode is desired when using their claim as a volume in a Pod. [See supported access modes by providers](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes)
+
+```shell
+# create a pvc
+dokku scheduler-kubernetes:add-pvc --name $NAME --access-mode $MODE --storage "100" --ns $NAMESPACE [--storage-class-name $CLASS]
+```
+
+Fields:
+    - `$NAME`: The name of the persistent volume clain
+    - `$MODE`: Access mode must be either of ReadWriteOnce, ReadOnlyMany or ReadWriteMany
+    - `--storage` takes a number and its in MB.
+    - `$NAMESPACE` : The namespace for the pvc.
+    - `$CLASS`: The storage class name.
+
+```shell
+# list pvcs
+dokku scheduler-kubernetes:list-pvc [$NAMESPACE]
+```
+
+```shell
+# delete pvc
+dokku scheduler-kubernetes:remove-pvc --name $NAME --ns $NAMESPACE
+```
+
+#### Mounting a volume using PVC
+Pods access storage by using the claim as a volume.
+
+```shell
+# mounting a volume (requires a re-deploy to take effect)
+dokku scheduler-kubernetes:mount $APP_NAME --name $VOLUME_NAME --claim-name $PVC_NAME --path /container/path
```

Comment:
Do any of these need to be flags? I would assume they should be arguments since they are all required.

Maybe we can set the claim name to be explicitly the same as the volume name?

Reply:
> Do any of these need to be flags?

No? Chose to use flags to make the argument order flexible

> Maybe we can set the claim name to be explicitly the same as the volume name?

Very good, idea. It will however mean you can't mount on that pvc anymore (which is not a bad idea as i have not been able to do 2 mounts on same pvc)

Reply:
If its not optional, it should be positional. Thats just how dokku works. Please change it so we can continue to have consistency.

Re: mounting twice, yeah lets change it so that they are the same. One less thing to configure, but also if you need the same mount twice, maybe you are making poor life choices.